src/ralph/lib/transitions/models.py

```diff
@@ -511,6 +511,7 @@ class Transition(models.Model):
     source = JSONField()
     target = models.CharField(max_length=50)
     actions = models.ManyToManyField('Action')
+    template_name = models.CharField(max_length=255, blank=True, default='')
```

Comment:
what will (should?) happen if I save template_name to DB and then remove it from settings?

Reply:
I will add some check for this which emit some warning.